Proton/Tests/Editor/EditorViewTests.swift

```diff
@@ -214,6 +215,7 @@ class EditorViewTests: XCTestCase {
 
         editor.replaceCharacters(in: .zero, with: testString)
         editor.insertAttachment(in: editor.textEndRange, attachment: attachment)
+        editor.layoutIfNeeded() // renders attachment within zero-sized Editor
```

Comment:
This is a smell to make the tests pass. It is expected since the size of the Editor is .zero, and I believe this "smell" is an acceptable trade off for production performance.